### PR TITLE
Did Till The Loops

### PR DESCRIPTION
**Pull request by Moyo75**
I completed the challenges til the loops section and also corrected some spelling errors in some of the readme files.

### CODE REVIEW

File: project-javascript-a1-introduction/JohnAdemoye.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <body>
+    <script>
+      // your code here
+
+      console.log(`Subjects \t Full Marks \t Pass Marks \t Obtained`);
+      console.log(`\n`);
+      console.log(`Math \t 100 \t 50 \t 80`);
+      console.log(`Science \t 100 \t 50 \t 60`);
+      console.log(`Computer \t 100 \t 50 \t 40`);
+      console.log(`Average \t ${(80 + 60 + 40) / 3}`);
```
Comment (basir) [inline project-javascript-a1-introduction/JohnAdemoye.html]: Good job

File: project-javascript-a1-introduction/README.md [hunk 2]
```diff
@@ -1,39 +1,42 @@
 # Project JavaScript A1 Introduction
-To code this project you need to know: 
+
+To code this project you need to know:
 Console, Comments, Data Types, Arithmetic Operators, String Concatenation, Properties, Methods, Built-in Objects
 
-If you are not familier with them download Learn JavaScript eBook from (https://codingwithbasir.com/learn-javascript)
+If you are not familiar with them download Learn JavaScript eBook from (https://codingwithbasir.com/learn-javascript)
```
Comment (basir) [inline project-javascript-a1-introduction/README.md]: nice

File: project-javascript-a1-introduction/README.md [hunk 3]
```diff
@@ -1,39 +1,42 @@
...
 
-| Subject  | Full Marks |Pass Marks |Obtained |
-| --- | --- | --- | --- |
-| Math  | 100  | 50  | 80  |
-| Science  | 100  | 50  | 60  |
-| Computer  | 100  | 50  | 40  |
-| Average  | 60  |
+3. Use console.log to log these info in the console.
 
+| Subject  | Full Marks | Pass Marks | Obtained |
```
Comment (basir) [inline project-javascript-a1-introduction/README.md]: good

File: project-javascript-b1-variables/JohnAdemoye.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <script>
+      // your code here
+
+      let annualCost,
```
Comment (basir) [inline project-javascript-b1-variables/JohnAdemoye.html]: you could use const for immutable variables.

File: project-javascript-b1-variables/README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Project JavaScript B1 Variables
 
-To code this project you need to know: Create a Variable: var
-, Create a Variable: let
+To code this project you need to know how to:
+Create a Variable: let
```
Comment (basir) [inline project-javascript-b1-variables/README.md]: thanks

File: project-javascript-c1-conditional-statements/JohnAdemoye.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <script>
+      // your code here
+
+      let Size, Energy, Height, Breed;
```
Comment (basir) [inline project-javascript-c1-conditional-statements/JohnAdemoye.html]: do not use let if you can use const

File: project-javascript-c1-conditional-statements/README.md [hunk 7]
```diff
@@ -2,13 +2,13 @@
 
 In this project you need to know :
 The if keyword,
-If...Else Statements Comparison Operators Logical Operators,
-Truthy and Falsy,
-Truthy and Falsy Assignment Ternary Operator,
-Else If Statements,
+if...else statements comparison operators logical operators,
```
Comment (basir) [inline project-javascript-c1-conditional-statements/README.md]: good

File: project-javascript-d1-functions/JohnAdemoye.html [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+      function nextYear(year) {
+        return year + 1;
+      }
+
+      console.log(formatDate(nextYear(year), month, day));
+
+      const formatDateConcised = (year, month, day) =>
+        year + '/' + convert2String(month) + '/' + convert2String(day);
+
+      console.log(formatDateConcised(year, month, day));
```
Comment (basir) [inline project-javascript-d1-functions/JohnAdemoye.html]: clean and perfect

File: project-javascript-d1-functions/README.md [hunk 9]
```diff
@@ -40,11 +40,11 @@ All variables are number.
 
 4. Declare a function named `showDayName` that get dayOfWeek (number) and log day name into the console. For example it gets `1` and log `Sunday`, `2` and log `Monday`, etc.
 
-5. Declare a function named `convert2String` that convert a number to string. If number is less that 10, it puts a 0 befor it. Example: `2` => `'02'`
+5. Declare a function named `convert2String` that converts a number to string. If number is less that 10, it puts a 0 before it. Example: `2` => `02`
```
Comment (basir) [inline project-javascript-d1-functions/README.md]: nice catch

File: project-javascript-e1-scope/README.md [hunk 10]
```diff
@@ -26,13 +26,13 @@ If you are not familier with them please download Learn JavaScript eBook at http
...
-4. Declare a function named `logColor1` without parameter and log `color` varibale into console.
+4. Declare a function named `logColor1` without parameter and log `color` variable into console.
 
-5. Declare a function named `logColor2` with a parameter named `color` and log `color` varibale into console.
+5. Declare a function named `logColor2` with a parameter named `color` and log `color` variable into console.
 
 6. Call function `logColor2` with `'red'` as argument and check the result.
 
-7. Declare a function named `logColor3` with a parameter named `color` and log `color` varibale into console.
+7. Declare a function named `logColor3` with a parameter named `color` and log `color` variable into console.
```
Comment (basir) [inline project-javascript-e1-scope/README.md]: thanks

File: project-javascript-g1-loops/README.md [hunk 11]
```diff
@@ -30,7 +30,7 @@ If you are not familier with them please download Learn JavaScript eBook from ht
 ```
 
 3. Create a for loop and log odd numbers from 1 to 11:
-4. Create a for loop and log event numbers from 11 to 1 in reverse:
+4. Create a for loop and log even numbers from 11 to 1 in reverse:
```
Comment (basir) [inline project-javascript-g1-loops/README.md]: nice

File: project-javascript-b1-variables/JohnAdemoye.html (hunk 4 above)
Comment (Moyo75) [reply project-javascript-b1-variables/JohnAdemoye.html]: OK. Thanks for that!

File: project-javascript-g1-loops/README.md (hunk 11 above)
Comment (Moyo75) [reply project-javascript-g1-loops/README.md]: Thanks!

File: project-javascript-e1-scope/README.md (hunk 10 above)
Comment (Moyo75) [reply project-javascript-e1-scope/README.md]: You're welcome.

File: project-javascript-d1-functions/README.md (hunk 9 above)
Comment (Moyo75) [reply project-javascript-d1-functions/README.md]: Thank you

File: project-javascript-d1-functions/JohnAdemoye.html (hunk 8 above)
Comment (Moyo75) [reply project-javascript-d1-functions/JohnAdemoye.html]: Nice, thanks!

File: project-javascript-c1-conditional-statements/README.md (hunk 7 above)
Comment (Moyo75) [reply project-javascript-c1-conditional-statements/README.md]: Thanks!

File: project-javascript-b1-variables/README.md (hunk 5 above)
Comment (Moyo75) [reply project-javascript-b1-variables/README.md]: You're welcome.

File: project-javascript-a1-introduction/README.md (hunk 2 above)
Comment (Moyo75) [reply project-javascript-a1-introduction/README.md]: Thanks!

File: project-javascript-a1-introduction/JohnAdemoye.html (hunk 1 above)
Comment (Moyo75) [reply project-javascript-a1-introduction/JohnAdemoye.html]: Thanks

**Pull request merged**